spec/schemas/inst_schema.json

```diff
@@ -430,6 +430,20 @@
           "description": "Path to the source file. Used by downstream tooling; not expected to be found in handwritten files",
           "type": "string"
         },
+        "all_bases_format": {
```

Comment:
@dhower-qc This is a new attribute, "all_bases_format", distinct from "format" (below), which is what Unique has been using thus far. What that intentional? :-) Do you want Unique to use this attribute instead of "format" when RV32/RV64 is required?